src/cmake/AxomConfig.cmake

```diff
@@ -71,3 +71,49 @@ configure_file(
 )
 
 install(FILES ${CMAKE_BINARY_DIR}/include/axom/config.hpp DESTINATION include/axom)
+
+#------------------------------------------------------------------------------
```

Comment:
Much of this logic was adapted from `conduit`'s installation logic.  Thanks @cyrush !